plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/editor.scss

```diff
@@ -0,0 +1,40 @@
+.wc-block-product-gallery-thumbnails {
+	// Disabled doesn't disable scroll.
+	&__scrollable {
+		overflow: hidden;
+	}
+	// Using Disabled from `@wordpress/components` breaks the layout
+	// as we're relying on flex.
+	&__thumbnail {
+		pointer-events: none;
+	}
+
+	// Editor uses :after pseudo-element to show outlines on hover or focus.
+	// This is a workaround to ensure the outline is visible even when the element is focused
+	// and we "override" :before pseudo-element so it's used as bottom and right gradient.
+	&::after {
+		opacity: 1 !important;
+		width: 100% !important;
+		height: 100% !important;
+		background: none !important;
```

Comment:
Have you considered increasing specificity instead of `important`?

Reply:
Considering it's in Editor and it's strongly expected to bring those values to the "editor defaults" I didn't. What I mean is, in this particular case I don't see much value in seeking stronger specificity, there's no argue or condition in which those values should be overwritten by even stronger specificity.